[2024-07-26T05:57:24.989+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-26T05:57:24.995+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_dag.minio_sensor manual__2024-07-26T05:57:23.678848+00:00 [queued]>
[2024-07-26T05:57:24.997+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_dag.minio_sensor manual__2024-07-26T05:57:23.678848+00:00 [queued]>
[2024-07-26T05:57:24.997+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-26T05:57:25.002+0000] {taskinstance.py:2330} INFO - Executing <Task(S3KeySensor): minio_sensor> on 2024-07-26 05:57:23.678848+00:00
[2024-07-26T05:57:25.004+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=466) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-26T05:57:25.005+0000] {standard_task_runner.py:64} INFO - Started process 467 to run task
[2024-07-26T05:57:25.006+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'minio_dag', 'minio_sensor', 'manual__2024-07-26T05:57:23.678848+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/7_minio_dag.py', '--cfg-path', '/tmp/tmp7x_g82t6']
[2024-07-26T05:57:25.007+0000] {standard_task_runner.py:91} INFO - Job 12: Subtask minio_sensor
[2024-07-26T05:57:25.038+0000] {task_command.py:426} INFO - Running <TaskInstance: minio_dag.minio_sensor manual__2024-07-26T05:57:23.678848+00:00 [running]> on host ecd5bbc03de3
[2024-07-26T05:57:25.109+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Bossixd' AIRFLOW_CTX_DAG_ID='minio_dag' AIRFLOW_CTX_TASK_ID='minio_sensor' AIRFLOW_CTX_EXECUTION_DATE='2024-07-26T05:57:23.678848+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-26T05:57:23.678848+00:00'
[2024-07-26T05:57:25.109+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-26T05:57:25.120+0000] {baseoperator.py:400} WARNING - S3KeySensor.execute cannot be called outside TaskInstance!
[2024-07-26T05:57:25.120+0000] {s3.py:117} INFO - Poking for key : s3://***/data.csv
[2024-07-26T05:57:25.125+0000] {base.py:84} INFO - Using connection ID 'minio_localhost' for task execution.
[2024-07-26T05:57:25.126+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio_localhost', conn_type='aws') credentials retrieved from login and ***.
[2024-07-26T05:57:25.415+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-07-26T05:57:25.416+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-26T05:57:25.420+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=minio_dag, task_id=minio_sensor, run_id=manual__2024-07-26T05:57:23.678848+00:00, execution_date=20240726T055723, start_date=20240726T055724, end_date=20240726T055725
[2024-07-26T05:57:25.433+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-26T05:57:25.440+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-26T05:57:25.440+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
